Code/Conviva-Examples/SharedCode/ViewController.swift

```diff
@@ -10,21 +10,38 @@ import UIKit
 import THEOplayerSDK
 import THEOplayerConnectorConviva
 
+#if canImport(THEOplayerTHEOliveIntegration)
+import THEOplayerTHEOliveIntegration
+#endif
+
 class ViewController: UIViewController {
     let player = THEOplayer(with: nil, configuration: nil)
     var conviva: ConvivaConnector?
     
+#if canImport(THEOplayerTHEOliveIntegration)
+    var THEOliveIntegration: THEOplayerTHEOliveIntegration.THEOliveIntegration?
+#endif
+    
     @IBOutlet weak var playerViewContainer: UIView!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        self.conviva = ConvivaConnector(configuration: convivaConfig, player: self.player)
         let playerView = PlayerView(player: player)
         playerView.translatesAutoresizingMaskIntoConstraints = true
         playerView.frame = playerViewContainer.bounds
         playerView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
         playerViewContainer.addSubview(playerView)
+        
+#if canImport(THEOplayerTHEOliveIntegration)
+        let THEOliveConfig = THEOliveConfiguration(externalSessionId: nil)
```

Comment:
I was not sure why this was needed because THEOplayer adds integrations automatically if it can find them.
But it is a good example to show how to add custom configs for integrations.